var/spack/repos/builtin/packages/flex/package.py

```diff
@@ -24,21 +24,29 @@
 ##############################################################################
 from spack import *
 
-
```

Comment:
I've changed the previous commit to correct this. 883d5c5 is now the most recent commit.